### DISCUSSION TITLE
The basic LaTeX functions do not display properly in my PDF.

### DISCUSSION THREAD

**Discussion by GuyNYAMSI** [quarto-dev/quarto-cli#9440] [Q&A] [answered]
### Description

project:
  type: book

book:

  title: "Rapport"

  author: 
    name: Guy 
  date: today

  chapters:
    - index.qmd
    - abstract.qmd
    - remerciement.qmd
    - lexique.qmd
    - intro.qmd
    - lesdonnees.qmd
    - methodologie.qmd
    - conclusion.qmd
    - references.qmd
    

bibliography: references.bib
author-title: "Auteur"
format:

  pdf:
    toc-title: 
      "Sommaire"

    pdf-engine: lualatex
    documentclass: scrreprt
    include-in-header: 
      text: |
        \usepackage{tikz}
        \usepackage{amsmath}
        \usepackage[T1]{fontenc}
        \usepackage{eurosym}
        \usepackage{a4} 
        \usepackage[frenchb]{babel}
        \usepackage{colortbl}
        \usepackage{pdfpages}
        \usepackage{graphics,color}
        \usepackage{amssymb}
        \usepackage[mathscr]{euscript}
        \usepackage{xspace}
        \usepackage{stmaryrd}
        \usepackage{xcolor}
        \usepackage{exscale}
        \usepackage{amsfonts}
        \usepackage{fullpage}
        \usepackage[linktocpage]{hyperref}
        \usepackage[all]{hypcap}
        \usepackage{longtable}
        \usepackage{multicol}
        \usepackage{multirow}
        \usepackage{rotating}
        \usepackage{wrapfig}
        \usepackage{url}
        \usepackage{wallpaper}
        \usepackage{makeidx}
        \usepackage{graphicx}
        \usepackage{array}
        \usepackage{tabularx}
        \usepackage{enumerate}
        \usepackage{pifont}
        \usepackage [dvips]{epsfig}
        \usepackage{fancyhdr}
        \textwidth=16.5cm \oddsidemargin=0pt \textheight=23.5cm
        \hoffset=0cm \voffset=0cm \evensidemargin=0pt \topmargin=0pt
        \headheight = 14pt \headsep = 20pt \marginparsep = 0pt
        \marginparwidth = 0pt \footskip = 25pt \marginparpush = 0pt
        \paperwidth = 21cm \paperheight = 29.7cm
        \linespread{1.2}% Mise en page des bordures
        %-----------------------------------------------------------------

        \pagestyle{fancy}

        % Ceci permet d'avoir les noms de chapitre et de section
        % en minuscules
        \renewcommand{\chaptername}{Partie}
        \renewcommand{\contentsname}{Sommaire}
        \renewcommand{\bibname}{Bibliographie}
        \renewcommand{\chaptermark}[1]{\markboth{#1}{}}
        \renewcommand{\sectionmark}[1]{\markright{\thesection\ #1}}
        \fancyhf{} % supprime les en-têtes et pieds pr\'ed\'efinis
        \fancyhead[L,R]{\bfseries\thepage}% Left Even, Right Odd
        \fancyhead[L]{\bfseries\rightmark} % Left Odd
        \fancyhead[R]{\bfseries\leftmark} % Right Even
        \fancyfoot[RE,RO]{\tiny \textbf{Actuariat}
        \textbf{\copyright}
        \textbf{ISFA 2023-2024}}
        \renewcommand{\headrulewidth}{1.pt}% filet en haut de page
        \addtolength{\headheight}{1.pt} % espace pour le filet
        \renewcommand{\footrulewidth}{1pt}% filet en bas de page
        %\addtolength{\footheight}{0.5pt} % espace pour le filet en bas
        \fancypagestyle{plain}{ % pages de tetes de chapitre
        %\fancyhead{} % supprime l'entete
        %\fancyfoot{} %supprime le pied de page
        \renewcommand{\headrulewidth}{1pt} % et le filet
        %\renewcommand{\footrulewidth}{0pt}
        }
        \newcommand{\clearemptydoublepage}{%
        \newpage{\pagestyle{plain}\cleardoublepage}}
        \rhead{\textbf{\thepage}} %{\textsl{\rightmark}}
        \lhead{\textsl{\leftmark}}%{\textbf{\thepage}}
        \lfoot{ \footnotesize{\tiny 
        \includegraphics[scale=0.05]{logo_isfa.png} 
        \includegraphics[scale=0.9]{LinkPact.jpg} 
        \includegraphics[scale=0.9]{lfm.png} 
        }}
        \usepackage{newlfont}
        \newlength{\defbaselineskip}
        \setlength{\defbaselineskip}{\baselineskip}
        \newcommand{\setlinespacing}[1]%
                  {\setlength{\baselineskip}{#1 \defbaselineskip}}
        \newcommand{\doublespacing}{\setlength{\baselineskip}%
                              {1 \defbaselineskip}}
        \newcommand{\singlespacing}{\setlength{\baselineskip}{\defbaseline
        skip}}

        %-----------------------------------------------------------------
        \usepackage{t1enc}
        \makeatletter
        \newcommand{\thechapterwords}
        { \ifcase \thechapter
        \or 1\or 2\or 3\or 4\or 5\or
          Six\or Sept\or Huit\or Neuf\or Dix\or Onze\fi}
        \makeatother

        \makeatletter
        \newcommand{\thickhrulefill}{\leavevmode \leaders \hrule height 1ex \hfill \kern \z@}
        \makeatother

        \makeatletter
        \def\@makechapterhead#1{%
          %\vspace*{50\p@}%
          \vspace*{15\p@}%
          {\parindent \z@ \centering \reset@font
              \thickhrulefill\quad
                \scshape \@chapapp{} \thechapterwords
                \quad \thickhrulefill
                \par\nobreak
                \vspace*{15\p@}%
                %\interlinepenalty\@
                \hrule
                \vspace*{15\p@}%
                \Huge \bfseries #1\par\nobreak
                \par
                \vspace*{15\p@}%
                \hrule
            \vskip 60\p@
            %\vskip 100\p@
          }}
        \makeatother

        \makeatletter
        \def\@makeschapterhead#1{%
          %\vspace*{50\p@}%
          \vspace*{15\p@}%
          {\parindent \z@ \centering \reset@font
                \thickhrulefill
                \par\nobreak
                \vspace*{15\p@}%
                \interlinepenalty\@M
                \hrule
                \vspace*{15\p@}%
                \Huge \bfseries #1\par\nobreak
                \par
                \vspace*{15\p@}%
                \hrule
            \vskip 60\p@
            %\vskip 100\p@
          }}
        \makeatother

        %-----------------------------------------------------------------
        \newtheorem{defi}{Définition}[section]
        \newtheorem{pro}{Proposition}[section]
        \newtheorem{cor}{Corollaire}[section]
        \newtheorem{lem}{Lemme}[section]
        \newtheorem{theo}{Théorème}[section]
        \newtheorem{remarque}{Remarque}[section]
        \newtheorem{propriete}{Propriété}[section]
        \newtheorem{ex}{\underline{Exemple}}[section]
        \newtheorem{cq}{Conséquence}[section]
        \newtheorem{act}{\underline{Activité}}[section]
        \newtheorem{exo}{\underline{Exercice}}[section]
        \frenchspacing

 #################################################################################
---
title: "Méthodologies statistiques pour la construction de la loi d'expérience"

crossref:
  #fig-title: Fig     
  #tbl-title: Tbl     
  title-delim: "&nbsp;:" 
  chapters: true    

---

## Présentation des différentes méthodes statistiques utilisées

Les approches Machine Learning sont de plus en plus utilisées dans les études de mortalité au détriment de l'approche statistique classique qui met l'accent sur l'inférence statistique et l'interpretabilité là où les méthodes de machine learning se concentrent sur la flexibilité et la performance prédictive. Le choix entre ces deux approches dépend très souvent des objectifs spécifiques de modélisation et des besoins en termes de précision et d'interprétabilité.

Dans le cadre de nos travaux, afin de construire des tables de mortalité prospectives, nous utiliserons les deux approches puis, nous définirons un critère bien précis afin de pouvoir comparer les deux approches.

## Aproche statistique classique 
 Les méthodes statistiques classiques, telles que les modèles de régression linéaire, les modèles de survie de Kaplan-Meier et les modèles de régression de Cox, fournissent des coefficients qui permettent d'interpréter directement l'effet de chaque variable explicative sur la mortalité. Cela permet aux assureurs de comprendre les facteurs qui influent sur la mortalité de leur portefeuille et d'ajuster leurs stratégies en conséquence. En assurance, il est souvent crucial de comprendre les risques associés à la mortalité afin de prendre des décisions de tarification et de gestion de portefeuille éclairées. Les modèles statistiques comme les
 modèles additifs généralisés permettent un contrôle plus direct et précis sur ces risques en fournissant des estimations précises de la mortalité et des facteurs de risque associés en définissant une relation pas nécéssairement linéaire entre l'espérance de la variable à expliquer $y$ représantant la mortalité et les variables explicatives $x$ représant les facteurs de risue associés.

### Les modèles additifs généralisés (GAM)
Introduit par Trevor Hastie et Robert Tibshirani dans leur ouvrage "Generalized Additive Models" publié en 1990, cette méthode est développé comme une extension des modèles linéaires généralisés (GLM) pour modéliser des relations non linéaires entre les variables explicatives $x$ et la variable réponse $y$.
Afin de comprendre les modèle GAM, revenons quelques peut sur les modèles GLM.

#### Le GLM

Dans un modèle de régression linéaire simple ou multiple ($y=\beta x$), on tente d'expliquer la valeur de la variable dépendante ou à expliquer par une combinaison linéaire d'autres variables  explicatives. Cette combinaison linéaire implique
que la variation de $y$ est directement proportionnelle à la variation de n'importe quelle variable explicative. De plus, le cadre statistique de la régression linéaire
où l'on souhaite construire des intervalles de confiance pour tester des hypothèses liées au modèle, suppose que $Y \sim \mathcal{N}(x\beta, \sigma^2)$ (Wood 2006, page 7 section 1.1.3).

L'idée que sous-entend le GLM est d'aléger ces hypothèses (relation directement proportionnelle et normalité de la distribution) afin d'introduire une fonction de lien 
permettant de modéliser une transformation de $\mathbb{E}(Y)$ et de généraliser les lois de
probabilités possible pour la variable aléatoire $Y$ à la famille exponentielle linéaire.
ainsi, l'équation du modèle linéaire de façon générale :

\begin{equation} 
y_i = \beta_0+ x_{i,1}\beta_1+\cdots+x_{i,p}\beta_p 
\end{equation}

s'écrit dans le cadre d'un GLM ainsi :

\begin{align}
g(\mu_i) &= \beta_0+ x_{i,1}\beta_1+\cdots+x_{i,p}\beta_p \label{equa1}\\
&= \mathbf{X_i\beta} 
\end{align}

où $y_i$ est la i-ème valeur de la variable à expliquer, $\mathbf{X_i}$ est i-ème ligne d'une matrice de design à $n$ observations formé de $p$ variables explicatives $x_j$, $j=1,\cdots,p$,  $\mathbf{\beta}$, un vecteur de $p+1$ paramètres à estimer et $\mu_i=\mathbb{E}(Y_i)$.

$g$ défini à l'équation $\eqref{equa1}$ est appelée \og fonction de lien\fg{}  et permet de définir la relation entre $\mu_i$ et le prédicteur linéaire défini par le membre de droite de l'équation $\eqref{equa1}$

## Aproche Machine Learning
Flexibilité : Les méthodes de machine learning, telles que les forêts d'arbres décisionnels, les réseaux de neurones, les modèles GAM (Generalized Additive Models), etc., sont flexibles et peuvent capturer des relations complexes entre les variables explicatives et la mortalité. Cela permet de modéliser des effets non linéaires et des interactions entre les variables.

Précision : Les modèles de machine learning peuvent potentiellement fournir des prédictions plus précises de la mortalité par rapport aux modèles traditionnels. Ils peuvent identifier des schémas subtiles dans les données qui pourraient être manqués par des méthodes plus simples.

Adaptabilité : Les modèles de machine learning peuvent s'adapter à différents types de données et structures, ce qui les rend adaptés à une variété de portefeuilles d'assurance et de situations.

Évolutivité : Les méthodes de machine learning peuvent être appliquées à de grands ensembles de données avec efficacité, ce qui les rend adaptées à l'analyse de grands portefeuilles d'assurance avec de nombreuses observations.

Flexibilité : Les méthodes de machine learning, telles que les forêts aléatoires, les réseaux de neurones, les modèles GAM, etc., offrent une plus grande flexibilité dans la modélisation des relations complexes entre les variables explicatives et la mortalité. Elles peuvent capturer des schémas non linéaires et des interactions complexes entre les variables.

Prédiction : Les méthodes de machine learning se concentrent souvent sur la prédiction de la mortalité plutôt que sur l'inférence statistique. Elles visent à fournir les meilleures prédictions possibles, même si elles ne permettent pas toujours d'interpréter directement les relations entre les variables.

Performance : Les méthodes de machine learning peuvent souvent offrir de meilleures performances prédictives que les approches statistiques classiques, en particulier lorsque les relations entre les variables sont complexes ou mal comprises.

Complexité : Les modèles de machine learning peuvent être plus complexes et nécessiter des compétences spécialisées pour les ajuster correctement et interpréter les résultats.

<img width="956" alt="code" src="https://github.com/quarto-dev/quarto-cli/assets/39628456/bcb6a4cf-b768-40ed-91c6-aec84b820ef4">

**Comment by mcanouil** [comment]
Could you share a small **self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

We cannot help with a document that big with that amount of modifications/additions which can conflict.

To debug your LaTeX document, you can add `keep-tex: true`, then look at the LaTeX code and render it manually to figure out what is causing your issue.

Additionally, could you properly format your post using code blocks for code and terminal outputs? Thanks.
If your code contains code blocks, you need to enclose it using more backticks, *i.e.*, usually four ` ```` `.

Finally, instead of adding a huge amount of LaTeX code in `include-in-header`, you should use a template or a partial: https://quarto.org/docs/journals/templates.html#overview



---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Reply by GuyNYAMSI** [reply, marked as answer]
Thank you, I found the source of the problem using  `keep-tex: true`
